### DISCUSSION TITLE
Wann wird die Bundeswallet für Kantone und Städte zur Verfügung stehen ?

### DISCUSSION THREAD

**Discussion by rschubnell** [e-id-admin/governance-sounding-board#52] [Q&A]
Wann wird die Bundeswallet für Kantone und Städte zur Verfügung stehen ? 
Ist ein Testflight Programm geplant um in den Kantonen Issuer und Verifier Komponenten erstellen zu können vor dem Go-Live der E-ID ? 
Wird die Wallet als OSS Software hier auf Git-Hub bereitgestellt ?
Wir stellen uns die Frage der Verfügbarkeit um unsere Ressourcenplanung der eigenen Wallet planen zu können.

**Comment by christianheimann** [comment]
In der Präsentation vom Partizipations-Meeting vom 4. Juli 2024 zur geplanten Public Beta (Sandbox 2.0) wurde detailliert ausgeführt, mit welchen Elementen 2025 geplant werden kann: https://youtu.be/S9Wdg575zgI?feature=shared&t=4545

Damit ist grundsätzlich eine Umgebung da, welche es erlauben wird, Issuer- und Verifier-Komponenten angehen zu können. Die Wallet wird dabei nicht nur als OSS bereitstehen, sondern als nutzbare App aus den offiziellen App-Stores.
